src/book/03-guides/01-main-codebase/02-script-tuner.mdx

```diff
@@ -0,0 +1,99 @@
+| <kbd>A</kbd>                   | Saves the script with a particular name. After pressing this key, write the name of the script and press <kbd>Enter</kbd>.                                                                                                                                                                                                                                                                           |
+| <kbd>R</kbd>                   | Manual refresh view.                                                                                                                                                                                                                                                                                                                                                                                 |
+| <kbd>M</kbd>                   | Mirrors the whole script.                                                                                                                                                                                                                                                                                                                                                                            |
+| <kbd>X</kbd>                   | Exits the help or exits gain editing.                                                                                                                                                                                                                                                                                                                                                                |
+| <kbd>CTRL</kbd> + <kbd>C</kbd> | Quits script tuner.                                                                                                                                                                                                                                                                                                                                                                                  |
+| <kbd>:help</kbd>               | Lists all commands that can be used in script tuner.                                                                                                                                                                                                                                                                                                                                                 |
+
+## List of Scripts
```

Comment:
```suggestion
## Common Scripts
```

Reply:
I'm not sure if this works. Also not sure of the type of 'common' being used. Common across robots or scripts that we use commonly/often?

Reply:
Was going for something like "most used", since there are be other scripts not listed here. If this is meant to be an exhaustive list though, then "List of Scripts" is definitely better.

Reply:
Ye, it's an exhaustive list.